test/UniswapConfig.test.ts

```diff
@@ -187,14 +185,14 @@ describe("UniswapConfig", () => {
     const tx9_ = await deployer.sendTransaction(tx9__);
     const tx9 = await tx9_.wait();
     expect(cfg9.underlying).to.equal(address(10));
-    expect(tx9.gasUsed).to.equal(23504);
```

Comment:
if-elses saving us some gas here 👍

Reply:
Bumped up the contract size slightly though :P (which was reduced by some of other other code removal changes)